mautic/mautic
# Adding Twig lint command to GH Actions to catch invalid TWIG syntax

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [y]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR is adding one more check for Twig Lint command into GH Actions so the linter reviews valid Twig file contents for us.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. There is not much to test. The main change is in the CI config file. You can see the new "Twig Lint" check bellow among PHPUNIT, Rector, PHPSTAN and otheres. 
3. The new check found one issue in the list of remote assets that require a plugin configuration so it wasn't discovered during testing of https://github.com/mautic/mautic/pull/11610
